Web/Controller/LandingPages.hs

```diff
@@ -17,7 +19,39 @@ instance Controller LandingPagesController where
 
     action ShowLandingPageAction { landingPageId } = do
         landingPage <- fetch landingPageId
+
+        -- Get all the Paragraphs for this Landing Page.
+        -- As we won't have a lot of Paragraphs, we can just fetch them all.
+        paragraphCtas <- query @ParagraphCta
```

Comment:
Here we query on by one. There's no big hit, as there aren't many paragraphs on each Landing page, but maybe there's a nicer way?